fdbrpc/sim2.actor.cpp

```diff
@@ -2093,11 +2100,14 @@ class Sim2 final : public ISimulator, public INetworkConnections {
 	// time is guarded by ISimulator::mutex. It is not necessary to guard reads on the main thread because
 	// time should only be modified from the main thread.
 	double time;
```

Comment:
I think a comment for each of these explaining the differences between them would be helpful